common/scripts/docker/installRedis.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash -e
+
+export COMPONENT="redis"
+export REDIS_DATA_DIR="$RUNTIME_DIR/$COMPONENT/data"
+export REDIS_CONFIG_DIR="$CONFIG_DIR/$COMPONENT"
+export SCRIPTS_DIR="$SCRIPTS_DIR"
+export REDIS_IMAGE="shipimg/rdsbase:master.698"
```

Comment:
there is a task in the main issue to use drydock images for all services once we get the basic installation working. so this can be deferred